backend/app/Http/Controllers/AssessmentController.php

```diff
@@ -50,7 +50,7 @@ public function createAssessment(Request $request)
 
             Assessment::create($data);
 
-            return $this->sendResponse(false, null, "assessment created.", $data, 200);
+            return $this->sendResponse(false, null, "assessment created.", $data, Response::HTTP_CREATED);
         } catch (\Exception $e) {
             return $this->sendResponse(true, 'something went wrong creating assessment', $e->getMessage(), null, 500);
```

Comment:
http response still missing here

Reply:
Which http response are you talking about? I think it is passed on line 53

Reply:
line 55

Reply:
Fixed